config/form-content/mandataris-edit/form.ttl

```diff
@@ -71,6 +71,21 @@ ext:fractieF
     sh:name "Fractie";
     sh:order 900;
     sh:path ( org:hasMembership org:organisation ).
+ext:linkTobesluitF
+    a form:Field;
+    form:displayType displayTypes:defaultInput;
+    sh:group ext:editMandatarisPG;
+    sh:name "Link naar besluit";
+    sh:order 900;
```

Comment:
I think this should have a different order than the field above.

Reply:
You are right! Changing it